### PR TITLE
Restructure optimize/wazuh folder

### PR DESCRIPTION
**Pull request by juankaromo**
Hi team,

This PR moves the wazuh.yml file to the /usr/share/kibana/optimize directory. 

In addition, it did a modification of the structure of this directory, in order to have all the resources of Wazuh in the same path.

This PR solves #2087 

The directory has now the following distribution:

```
/usr/share/kibana/optimize
└── wazuh/
    ├── logs/
    |    ├── wazuhapp.log
    |    └── wazuhapp-plain.log
    ├── downloads/
    |    └── reports/
    └── config/
         └── wazuh.yml
```

Regards,

### CODE REVIEW

File: server/lib/initial-wazuh-config.js [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+#hosts:
+#  - <id>:
+#     url: http(s)://<url>
+#     port: <port>
+#     user: <user>
+#     password: <password> 
+
+hosts:
+  - default:
+     url: http://localhost
```
Comment (jsanchez91) [inline server/lib/initial-wazuh-config.js]: Set the protocol as `https`

File: server/logger.js [hunk 2]
```diff
@@ -37,7 +37,7 @@ const initLogger = () => {
       new winston.transports.File({
         filename: path.join(
           __dirname,
-          '../../../optimize/wazuh-logs/wazuhapp.log'
+          '../../../optimize/wazuh/logs/wazuhapp.log'
```
Comment (jsanchez91) [inline server/logger.js]: The log folder path is repeated a lot, maybe you can save it to a constant. 
```js
const logsPath = '../../../optimize/wazuh/logs';

path.join(__dirname, `${logsPath}/wazuhapp.log`);
```

File: public/controllers/settings/components/logs.js [hunk 3]
```diff
@@ -98,7 +98,7 @@ export default class SettingsLogs extends Component {
           <EuiFlexGroup>
             <EuiFlexItem>
               <EuiText color="subdued" style={{ paddingBottom: '15px' }}>
-                Log file located at /usr/share/kibana/optimize/wazuh-logs/wazuhapp.log
+                Log file located at /usr/share/kibana/optimize/wazuh/logs/wazuhapp.log
```
[the remark this answers is not in the document]
Comment (juankaromo) [reply public/controllers/settings/components/logs.js]: This is not within the scope of this issue. This was before this restructuring, so we should open issue to study it. But as far as this issue is concerned, everything is correct.

File: server/logger.js (hunk 2 above)
Comment (juankaromo) [reply server/logger.js]: Here if you are right, please change `../../optimize/wazuh/logs`  with a constant.

**Comment by juankaromo** [comment]
LGTM, but we need to take into account that the removal plugin step in the upgrade guide (README.md) needs to be reviewed.

**Pull request merged**